compiler/src/iree/compiler/Dialect/Flow/Transforms/RegionOpUtils.cpp

```diff
@@ -319,9 +347,19 @@ FailureOr<Flow::DispatchRegionOp> Flow::movePrecedingOpIntoDispatchRegion(
 FailureOr<Flow::DispatchRegionOp> Flow::wrapOpInDispatchRegion(
     RewriterBase &rewriter, Operation *op,
     Optional<Flow::WorkloadBuilder> workloadBuilder) {
-  Optional<ValueRange> workload = std::nullopt;
-  if (workloadBuilder.has_value()) workload = workloadBuilder->workload;
-  // Make an empty dispatch region right before the op.
+  Optional<SmallVector<Value>> workload = std::nullopt;
```

Comment:
ditto, the check is happening at l.357 before using it. I think default can be empty list, and it is valid?